### PR TITLE
Update popdata stateless validation

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: src/vbk/pop_common.cpp [hunk 1]
```diff
@@ -16,12 +16,6 @@ altintegration::PopContext& GetPop()
     return *app;
 }
 
-void StopPop()
-{
-    if (app == nullptr) return;
-    app->stop();
-}
-
```
Comment (Warchant) [inline src/vbk/pop_common.cpp:24]: revert this change. 
call `shutdown` instead of stop

**Review by Warchant — CHANGES_REQUESTED** [review]

File: src/vbk/pop_common.cpp (hunk 1 above)
Comment (Mr-Leshiy) [reply src/vbk/pop_common.cpp:24]: FIxed

**Review by Warchant — APPROVED** [review]
